### DISCUSSION TITLE
displaying  oracle data as a table at grafana

### DISCUSSION THREAD

**Discussion by hakanozanlagan** [grafana/grafana-infinity-datasource#623] [General]
hi,
i am trying to use infinity datasource to display a zabbix item value as a table
I created a host in zabbix and using zabbix oracle get function to get data in json format into an item.  then I created data source in grafana and use zabbix api to get item value.
zabbix response is below
```json
{
  "jsonrpc": "2.0",
  "result": [
    {
      "value": [
        {
          "COMPID": "123123",
          "COMPANY": "COMPA1",
          "SUCCESS": "7"
        },
        {
          "COMPID": "32131233",
          "COMPANY": "COMPA2",
          "SUCCESS": "84"
        },
        {
          "COMPID": "102218372",
          "COMPANY": "UNITEDPAY",
          "SUCCESS": "4"
        }
      ]
    }
  ],
  "id": 2
}
```
item value is a json and contains table data. but grafana shows this json value as a single cell and it's normal

is there a way to use only this value as json input for  create a table?

**Comment by hakanozanlagan** [comment]
ok i used below  jsonpath command in query parsing and it's working now.

$.result[*]

**Reply by fsantosvwfs** [reply]
where do you put that to get value json to be recognized?
